Formula/j/julia.rb

```diff
@@ -193,6 +193,9 @@ def install
     ]
 
     assert_equal "4", shell_output("#{bin}/julia #{args.join(" ")} --print '2 + 2'").chomp
+    # Tests below time out on 12-x86_64.
+    return if OS.mac? && Hardware::CPU.intel? && MacOS.version <= :monterey
+
     system bin/"julia", *args, "--eval", 'Base.runtests("core")'
```

Comment:
I think the `Base.runtests` is the longest of the bunch, by far. Should we simply skip that one on Monterey, and still test loading of libraries below?

Reply:
Sure, that's fine. I'm a bit wary of testing order though, so I'm not going to move `Base.runtests` (even if it'll be cleaner to read if moved last with an early `return`).